_data/simple-icons.json

```
@@ -3707,6 +3707,11 @@
             "hex": "EC1C24",
             "source": "https://facepunch.com/img/brand/default-light.svg"
         },
+        {
+            "title": "Fairphone",
+            "hex": "2FA6DF",
+            "source": "https://www.fairphone.com/"
```

```suggestion
            "source": "https://www.fairphone.com"
```
Remove trailing slash